README.md

```diff
@@ -10,16 +10,16 @@ It can be found at `https://twinery.org/cookbook/`.
 
 ## Building the Cookbook using MkDocs
 
-The *Cookbook* can be built using Python. Run `pip install -r requirements.txt` to install the requirements and then using `mkdocs build` to build or `mkdocs serve` to create a build and host a local server version.
+The *Cookbook* can be built using Python. Run `pip install -r requirements.txt` to install the requirements.
 
-**Note:** Depending on how Python is installed in Windows, users may need to use `pip install --user -r requirements.txt` to install and then use `python -m` before any MkDocs commands to build or serve.
+* `mkdocs build` creates HTML and CSS files based on the markdown files.
+* `mkdocs serve` builds and hosts a local server version.
 
-## *Want to suggest a change or new example?*
+**Note:** Depending on how Python is installed in Windows, users may need to use `pip install --user -r requirements.txt` to install and then use `python -m` before any MkDocs commands.
 
-Start with the [Code of Conduct](https://github.com/iftechfoundation/twine-cookbook/blob/master/CODE_OF_CONDUCT.md). Then, include as much as possible of the following in an [issue](https://github.com/iftechfoundation/twine-cookbook/issues):
+## *Want to suggest a change or new example?*
 
-* A short summary of the example(s) or suggestions(s)
-* How (or if) you would like to be recognized for your contribution
+Start with the [Code of Conduct](https://github.com/iftechfoundation/twine-cookbook/blob/master/CODE_OF_CONDUCT.md) and then review the steps in the [Contributing guide](https://github.com/videlais/twine-cookbook/blob/master/CONTRIBUTING.md).
```

Comment:
Shouldn't this URL be https://github.com/iftechfoundation/twine-cookbook/blob/master/CONTRIBUTING.md?
Or rather, `./CONTRIBUTING.md`? A relative path would be better.

Reply:
Fixed